indico/modules/users/client/js/react/components/ICSCalendarLink.jsx

```diff
@@ -0,0 +1,135 @@
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+import {Button, Dropdown, Icon, Input, Label, Grid, Popup, Header} from 'semantic-ui-react';
+
+import signURL from 'indico-url:core.sign_url';
```

Comment:
I think in other files we have those imports on top. also, missing file header. `python bin/maintenance/update_header.py indico` should tell you about all files with missing headers

Reply:
We should run that as part of the commit hook.

Reply:
```
$ time python bin/maintenance/update_header.py indico
Updating headers to the year 2019 for all git-tracked files...
python bin/maintenance/update_header.py indico  1.93s user 0.04s system 99% cpu 1.977 total
```

Not sure whether I like that idea. CI is probably the better place for it.